src/main/webapp/partials/instructorHelpStudents.jsp

```diff
@@ -0,0 +1,609 @@
+                  <div class="col-sm-11">
+                    <p class="form-control-static">Team A</p>
+                  </div>
+                </div>
+                <div class="form-group">
+                  <label class="col-sm-1 control-label">Official Email Address:</label>
+                  <div class="col-sm-11">
+                    <p class="form-control-static">alice@email.com</p>
+                  </div>
+                </div>
+                <div class="form-group">
+                  <label class="col-sm-1 control-label">Comments:</label>
+                  <div class="col-sm-11">
+                    <p class="form-control-static">Alice is a transfer student.</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <p>
+            The student's profile page displays the student's details and course-related information, such as:
+          </p>
+          <ul>
+            <li>
+              <b>Section name</b>: the name of the section you enrolled the student in. This only appears if sections are created for the course.
+            </li>
+            <li>
+              <b>Team name</b>: the name of the team you enrolled the student in, or <i>NA</i> if the student does not belong to a team.
+            </li>
+            <li>
+              <b>Official email address</b>: the email address that will be used to contact the student, taken from enrollment information
+            </li>
+            <li>
+              <b>Comments</b>: additional student information you entered in the Comments column during enrollment
+            </li>
+          </ul>
+          <p>
+            Below this is the <b>More Info</b> section containing a personal description given by the student, if any.<br>
+            You can press the <span class="text-muted glyphicon glyphicon-resize-full"></span> button in the top-right corner to display the information in a lightbox for better readability.
+          </p>
+          <div class="bs-example">
+            <div class="modal fade" id="studentProfileMoreInfo" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+              <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title">Alice Betsy's Profile - More Info</h4>
+                  </div>
+                  <div class="modal-body">
+                    <br>
+                    <%-- Note: When an element has class text-preserve-space, do not insert HTML spaces --%>
+                    <p class="text-preserve-space height-fixed-md">
+                      Hi I am Alice Betsy! I am from Colorado, America. I am a programming and gaming enthusiast. Aspiring to become a Software Architect in a well reputed organization.
```

Comment:
The added HTML spaces make the info look weird: 
<img width="916" alt="screen shot 2018-05-06 at 8 42 55 pm" src="https://user-images.githubusercontent.com/6811428/39673395-17ee63ce-516e-11e8-809a-2997928232b4.png">